### PR TITLE
Feature/rails3

### PR DESCRIPTION
**Pull request by tessi**
Let the plug-in work with the openproject rails3 version

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,28 +14,27 @@ This plugin aims to be compatible with
...
-== Development
+Development
+-----------
 
 To run the tests, you will need a current version of our Dev Tools
-[https://github.com/finnlabs/redmine-dev-tools] and
-[https://github.com/finnlabs/chiliproject_cucumber]. After running
+* [Redmine Dev Tools](https://github.com/finnlabs/redmine-dev-tools and)
+* [Chiliproject Cucumber](https://github.com/finnlabs/chiliproject_cucumber).
+
```
Comment (tessi) [inline README.md]: Both, dev-tools and chiliproject_cucumber, are deprecated for rails3 usage. They should not be used anymore (as they are integrated in the rails3 branch).

File: lib/openproject_help_link/version.rb [hunk 2]
```diff
@@ -0,0 +1,3 @@
+module HelpLink
+  VERSION = "2.0.0"
```
Comment (tessi) [inline lib/openproject_help_link/version.rb]: We have a version 2.0.1 in the Changelog file and a Version 2.0.0 here. That doesn't match ;)

BTW: Isn't a version raise to 3.0 more appropriate, as we introduce non backwards compatible changes and support rails3 now?

File: openproject_help_link.gemspec [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  s.version     = HelpLink::VERSION
+  s.authors     = ["Finn GmbH"]
+  s.email       = ["info@openproject.org"]
+  s.homepage    = "http://www.openproject.org"
+  s.summary     = "Help link targettarget  configuration"
+  s.description = "This plugin allows you to change the target of the help link
+                   in your OpenProject. Doing so allows you to e.g. have a
+                   support project within your OpenProject."
+
+  s.files = Dir["{app,config,db,lib}/**/*"] + ["GPL", "Rakefile", "README.rdoc"]
```
Comment (tessi) [inline openproject_help_link.gemspec]: README.rdoc was renamed to README.md - the wrong filename produces an invalid gemspec

File: test/openproject_help_link_test.rb [hunk 4]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class HelpLinkTest < ActiveSupport::TestCase
+  test "truth" do
+    assert_kind_of Module, HelpLink
```
Comment (tessi) [inline test/openproject_help_link_test.rb]: This is a pretty sad test suite :unamused:

File: README.md [hunk 5]
```diff
@@ -12,28 +14,27 @@ This plugin aims to be compatible with
 If you are running into compatibility issues, please report a bug in the
 project's issue tracker.
 
-== Installation
+Installation
+------------
```
Comment (tessi) [inline README.md:18]: We should explain that the user needs to edit the Gemfile.local of his openproject installation. And and this gem (giving him the appropriate gem line. For example <code>gem 'openproject_help_link', :git => 'git@github.com:finnlabs/help_link.git'</code>

File: README.md [hunk 6]
```diff
@@ -1,9 +1,11 @@
...
+=====================
 
 This plugin allows you to change the target of the help link in your
 OpenProject. Doing so allows you to e.g. have a support project within your OpenProject.
 
-== Requirements
+Requirements
+------------
 
 This plugin aims to be compatible with
```
Comment (tessi) [inline README.md:10]: This is outdateded. We want to be compatible with the rails3 branch now.

**Comment by tessi** [comment]
I'm not sure about how to execute the cukes. Maybe @linki has some insight here?

**Comment by tessi** [comment]
Our files should begin with the GPL header. See: https://www.gnu.org/licenses/gpl.html#howto

**Comment by cratz** [comment]
I think we should NOT merge this in the master as long as we run Rails2 apps. For Rails3 apps we can use the branch.

**Comment by sschu** [comment]
I guess we can close this.

**Pull request closed without merging**